src/canvas/SelectableCanvas.ts

```diff
@@ -949,17 +930,16 @@ export class SelectableCanvas<
 
     // if we hit the corner of an activeObject, let's return that.
     if (
-      // ts doesn't get that if shouldLookForActive is true, activeObject exists
+      // ts doesn't get that if aObjects has one object, activeObject exists
       activeObject &&
-      shouldLookForActive &&
+      aObjects.length >= 1 &&
       activeObject._findTargetCorner(pointer, isTouch)
     ) {
       return activeObject;
     }
     if (
       aObjects.length > 1 &&
-      isActiveSelection(activeObject) &&
-      !skipGroup &&
+      activeObject === this._activeSelection &&
```

Comment:
this condition is a bit silly because if aObjects > 1 we are dealing with the active selection but I think you put that for TS